manager/dispatcher/dispatcher_test.go

```diff
@@ -575,8 +608,8 @@ func TestAssignmentsSecretUpdateAndDeletion(t *testing.T) {
 	assert.NoError(t, err)
 
 	// FIXME(aaronl): This is hard to maintain.
```

Comment:
I found the magic numbers (such as "17" below in the old version) difficult to understand and keep up to date. In the past, when I made changes to the set of task states, it would break these tests and it was hard to understand how to fix them correctly.